experimental/cmd/dive/app.go

```
@@ -2021,24 +2149,20 @@ func (a *App) Run() error {
+	return inline.Run(a)
 }
 
-// printIntroToScrollback prints the intro/splash screen to scrollback.
+// printIntroToScrollback prints the intro/splash screen before the event loop
+// starts, when the runner cannot print for us yet.
 func (a *App) printIntroToScrollback() {
-	view := a.buildIntroView()
-	tui.Print(view)
+	i := a.appendIntro()
+	a.messages[i].emitted = true
+	tui.Print(a.introView(a.messages[i]))
```

golangci-lint (2.13.2)

[error] Error return value of `tui.Print` is not checked

(errcheck)